app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitSubmittersApiController.php

```diff
@@ -0,0 +1,322 @@
+                    'email' => ['=@', '@@', '=='],
+                    'id' => ['=='],
+                    'full_name' => ['=@', '@@', '=='],
+                    'member_id' => ['=='],
+                    'member_user_external_id' => ['=='],
+                    'has_accepted_presentations' => ['=='],
+                    'has_alternate_presentations' => ['=='],
+                    'has_rejected_presentations' => ['=='],
+                    'presentations_track_id' => ['=='],
+                    'presentations_selection_plan_id' => ['=='],
+                    'presentations_type_id' => ['=='],
+                    'presentations_title' => ['=@', '@@', '=='],
+                    'presentations_abstract' => ['=@', '@@', '=='],
+                    'presentations_submitter_full_name' => ['=@', '@@', '=='],
+                    'presentations_submitter_email' => ['=@', '@@', '=='],
+                    'is_speaker' => ['=='],
+                ];
+            },
+            function () {
+                return [
+                    'first_name' => 'sometimes|string',
+                    'last_name' => 'sometimes|string',
+                    'email' => 'sometimes|string',
+                    'id' => 'sometimes|integer',
+                    'full_name' => 'sometimes|string',
+                    'member_id' => 'sometimes|integer',
+                    'member_user_external_id' => 'sometimes|integer',
+                    'has_accepted_presentations' => 'sometimes|string|in:true,false',
+                    'has_alternate_presentations' => 'sometimes|string|in:true,false',
+                    'has_rejected_presentations' => 'sometimes|string|in:true,false',
+                    'presentations_track_id' => 'sometimes|integer',
+                    'presentations_selection_plan_id' => 'sometimes|integer',
+                    'presentations_type_id' => 'sometimes|integer',
+                    'presentations_title' => 'sometimes|string',
+                    'presentations_abstract' => 'sometimes|string',
+                    'presentations_submitter_full_name' => 'sometimes|string',
+                    'presentations_submitter_email' => 'sometimes|string',
+                    'is_speaker' => 'sometimes|string|in:true,false'
+                ];
+            },
+            function () {
+                return [
+                    'id',
+                    'first_name',
+                    'last_name',
+                    'full_name',
+                    'email',
+                    'created',
+                    'last_edited',
+                ];
+            },
+            function ($filter) use ($summit) {
+                if ($filter instanceof Filter) {
+                    $filter->addFilterCondition(FilterElement::makeEqual('summit_id', $summit->getId()));
+                }
+                return $filter;
+            },
+            function () {
+                return IMemberSerializerTypes::Submitter;
```

Comment:
this serializer does not serve purposes of the CSV
please take a look at speakers CSV counter part

Reply:
fixed